.github/workflows/run-integ-tests.yaml

```diff
@@ -123,9 +123,6 @@ jobs:
           - # issue #49
             app_to_test: ts-eks-helm
             mode: upgrade
```

Comment:
I assume this one is still broken because of #48? (If so, let's update line 123 just for completeness)

Reply:
well its broken because our current release branch is broken mainly. So we need 2 releases then can enable this and the helm one.